spec/features/incomes/creates_new_income.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+feature "creates new income" do
+  scenario "successfully" do
+    user = create(:user)
+    create(:budget, user: user)
+    log_in_as user
+
+    visit new_income_path
+    fill_in "income_name", with: "Salary"
+    select "1", from: "income_due_date_3i"
+    select "April", from: "income_due_date_2i"
+    select "2017", from: "income_due_date_1i"
```

Comment:
A question: should there be a validation against dates in the past?

Reply:
There should be, for both incomes and expenses, thanks for the pick up, I will add in another PR